src/main/java/com/google/sps/data/DBFuture.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.meltingpot.data;
+
+import com.google.api.core.ApiFuture;
+import java.util.concurrent.ExecutionException;
+
+public class DBFuture {
```

Comment:
I think this should be called DBUtil. The method can be called getFuture or something. I'd expect we have more util functions for DB at some point.
Actually this is also a good place to find all the constants for your collection names and stuff.

Reply:
Agree, might want to just collapse DBRefs and DBFuture

Reply:
Makes sense, done in 3e85a01